db/ringapp/property/PROP_000002/metaproperties.yaml

```diff
@@ -20,11 +20,11 @@ METAPROP_000003:
   citation: []
   commutative_only: false
   crosstable_pk: 168
-  example: null
+  example: RING_000002
   has_metaproperty: false
   metaproperty: METAPROP_000003
   property: PROP_000002
-  relation: 1
+  relation: null
```

Comment:
I'm not sure what's going on here: why is the relation field being nulled and replaced with an example?  Metaprop 3 is "subring" so ideally it'll be disproven by a relation.

Reply:
Yes, this PR is based on your [latest](https://github.com/rschwiebert/dart_data/commit/461225a38d01c6685b3217f8ab0067de57be0486) commit.